src/SFA.DAS.Commitments.Api.IntegrationTests/DatabaseSetup/CommitmentsDatabase.cs

```diff
@@ -0,0 +1,137 @@
+                "EffectiveToDate"
+            });
+        }
+
+        public async Task InsertCommitments(IEnumerable<DbSetupCommitment> commitments)
+        {
+            await BulkInsertRows(commitments, CommitmentTableName, new[]
+            {
+                "Id", "Reference", "EmployerAccountId", "LegalEntityId", "LegalEntityName",
+                "LegalEntityAddress", "LegalEntityOrganisationType", "ProviderId",
+                "ProviderName", "CommitmentStatus", "EditStatus", "CreatedOn",
+                "LastAction", "LastUpdatedByEmployerName", "LastUpdatedByEmployerEmail",
+                "LastUpdatedByProviderName","LastUpdatedByProviderEmail"
+            });
+        }
+
+        public async Task InsertDataLockStatuses(IEnumerable<DbSetupDataLockStatus> dataLockStatuses)
+        {
+            await BulkInsertRows(dataLockStatuses, DataLockStatusTableName, new[]
+            {
+                "Id", "DataLockEventId", "DataLockEventDatetime", "PriceEpisodeIdentifier",
+                "ApprenticeshipId", "IlrTrainingCourseCode", "IlrTrainingType",
+                "IlrActualStartDate", "IlrEffectiveFromDate", "IlrPriceEffectiveToDate",
+                "IlrTotalCost", "ErrorCode", "Status", "TriageStatus",
+                "ApprenticeshipUpdateId", "IsResolved", "EventStatus", "IsExpired",
+                "Expired"
+            });
+        }
+
+        public async Task<long> GetEmployerId(long apprenticeshipId)
+        {
+            using (var connection = new SqlConnection(DatabaseConnectionString))
+            {
+                await connection.OpenAsync();
+                // no need for param, as apprenticeshipId comes from test code not user
+                using (var command = new SqlCommand(
+                    $@"select EmployerAccountId from dbo.Commitment c
+                    join dbo.Apprenticeship a on c.Id = a.CommitmentId
+                    where a.Id = {apprenticeshipId}", connection))
+                {
+                    return (long)await command.ExecuteScalarAsync();
+                }
+            }
+        }
+
+        public async Task<T> GetJobProgress<T>(string columnName)
+        {
+            using (var connection = new SqlConnection(DatabaseConnectionString))
+            {
+                await connection.OpenAsync();
+                using (var command = new SqlCommand($"SELECT {columnName} FROM [dbo].[JobProgress]", connection))
+                {
+                    var result = await command.ExecuteScalarAsync();
+                    //todo: LastId returns DbNull, this just returns null
+                    //https://stackoverflow.com/questions/7927211/executescalar-returns-null-or-dbnull-development-or-production-server
+                    //ExecuteScalar returns DBNull for null value from query and null for no result
+                    return result == DBNull.Value ? default(T) : (T)result;
```

Comment:
can be simplified to 

`return result == DBNull.Value ? default : (T)result;`

Reply:
Switched to using the default literal as suggested.